session_2/student_registry/db/migrate/20210114031001_add_fields_to_students.rb

```diff
@@ -0,0 +1,8 @@
+class AddFieldsToStudents < ActiveRecord::Migration[6.1]
+  def change
+    add_column :students, :branch, :string
+    add_column :students, :cgpa, :decimal, precision:2, scale: 2
```

Comment:
Check if you can store `10.00` as the CGPA - it's definitetly a valid cgpa, right?

Reply:
Precision is 2, it should be able to store right?